app/code/Magento/Checkout/etc/adminhtml/system.xml

```diff
@@ -48,6 +48,10 @@
                     <label>Show Cross-sell Items in the Shopping Cart</label>
                     <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
                 </field>
+                <field id="enable_clear_shopping_cart" translate="label" type="select" sortOrder="20" showInDefault="1" showInWebsite="1" showInStore="1" canRestore="1">
```

Comment:
This only needs to be configurable at the *website* level.  We can remove `showInStore` for this

Reply:
Removed `showInStore`